ext/curl/interface.c

```diff
@@ -2256,6 +2260,9 @@ static int _php_curl_setopt(php_curl *ch, zend_long option, zval *zvalue) /* {{{
 #endif
 #if LIBCURL_VERSION_NUM >= 0x073d00 /* Available since 7.61.0 */
 		case CURLOPT_DISALLOW_USERNAME_IN_URL:
+#endif
+#if LIBCURL_VERSION_NUM >= 0x074000 /* Available since 7.64.0 */
+		case CURLOPT_DISALLOW_USERNAME_IN_URL:
```

Comment:
This looks wrong; should this be `CURLOPT_HTTP09_ALLOWED` instead?

Reply:
Impressive to get such a simple change wrong, you are absolutely right. I fixed it my fork, it looks like github automatically added the second commit to this pull request?! This is my first pull request on github, so if something else is needed, please let me know.

Reply:
> I fixed it my fork, it looks like github automatically added the second commit to this pull request?!

Yes, as soon as you modify your branch, the PR gets synced.